### ISSUE TITLE
Compile Issues  pkg.refresh_db result

### ISSUE THREAD

**Issue by damon-atkins** [saltstack/salt-winrepo-ng#648]
```
salt-winrepo-ng\check-mk-agent.sls:
    - failed to compile, check syntax, Conflicting ID 'Not Found'
salt-winrepo-ng\duplicati.sls:
    - pkgname "duplicati", version "skip_urltest" is not defined as a dictionary(hash) key
salt-winrepo-ng\intellij.sls:
    - failed to compile, check syntax, Conflicting ID '1'
```

Cheers

**Comment by myii** [comment]
@damon-atkins I think these are resolved now after issue #874, PR #897.  Is that right?

**Comment by damon-atkins** [comment]
I had forgot about this.....   I will close it
